test/suite/test_tiered19.py

```diff
@@ -116,6 +116,26 @@ def test_gcp_filesystem(self):
         self.assertRaisesHavingMessage(wiredtiger.WiredTigerError,
             lambda: ss.ss_customize_file_system(
                 session, no_access_bucket, None, self.get_fs_config(prefix)), err_msg)
+        
+         # We cannot use the file system to create files, it is readonly.
+        # So use python I/O to build up the file.
+        f = open('foobar', 'wb')
+
+        outbytes = ('MORE THAN ENOUGH DATA\n'*100000).encode()
+        f.write(outbytes)
+        f.close()
+
+        # Flushing copies the file into the file system.
+        self.assertEquals(ss.ss_flush(session, fs, 'foobar', 'foobar', None), 0)
+        self.assertEquals(ss.ss_flush_finish(session, fs, 'foobar', 'foobar', None), 0)
```

Comment:
Do we have a delete test coming in the future? I assume that running this 100 times will result in 100 files left in the GCP bucket. 
I'm also curious about what we do if the the file already exists in the bucket, but that my be out of scope for this ticket.

Reply:
Each test should have a random prefix generated so we _shouldn't_ have an identical file in the bucket.
I am not sure how to handle deletion since we do not want to be deleting any files in the cloud in storage source class.

Reply:
Sounds good 👍 
I was more worried about leaving orphan files around and the bucket growing in size over time, but the changes in WT-10359 mention a lifecycle rule on the gcp bucket that auto cleans up old files that will fix this for us.